lib/matplotlib/tests/test_pickle.py

```diff
@@ -5,11 +5,15 @@
 import numpy as np
 import pytest
 
+import pickle
```

Comment:
this is already imported just above; I'm actually surprised pycodestyle doesn't warn about it... and BytesIO too